src/configuration/formatting-evaluator-config.ts

```diff
@@ -61,6 +61,26 @@ const rewardsType = Type.Object(
 
 export const formattingEvaluatorConfigurationType = Type.Object(
   {
+    /**
+     * Configuration for readability scoring using Flesch-Kincaid
+     */
+    readabilityScoring: Type.Object(
+      {
+        enabled: Type.Boolean({
+          default: true,
+          description: "Enable Flesch-Kincaid readability scoring",
+        }),
+        weight: Type.Number({
+          default: 0.3,
+          description: "Weight of readability score in the total formatting score",
+        }),
+        idealScore: Type.Number({
```

Comment:
/ask how exactly is idealScore used in the implementation? Are these the best config options? If so should any be renamed for clarity?

Reply:
@shiv810 i agree with `targetReadabilityScore` please fix